grammar.hgr

```diff
@@ -49,6 +51,8 @@ grammar {
       javascript: "'([^\\\\\\'\\n]|\\\\[\\\"\\'nrbtfav\\\\?]|\\\\[0-7]{1,3}|\\\\x[0-9a-fA-F]+|\\\\[uU]([0-9a-fA-F]{4})([0-9a-fA-F]{4})?)*'"
     } -> wdl_unescape(:string)
 
+    r'else' -> :else
+    r'then' -> :then
```

Comment:
Actually suspect this wouldn't be necessary... but if it is, then it's probably also required in the output expression section too